README.md

```diff
@@ -20,13 +20,43 @@ defp deps do
 end
 ```
 
+For Elixir 1.13 and later, modify `.formatter.exs` by adding
```

Comment:
we should add a section (or move it from where ever it is) for elixir <= 1.12 here if we're planning to keep mix surface.format around while waiting for folks to upgrade to elixir 1.13

Reply:
i like the format you have under the usage section for doing that, ie:

```markdown
## Installation
// add dep

### Elixir 1.13 and later
// what you have here

### Elixir 1.12 and prior
// the original instructions
```

Reply:
@lnr0626 I was going to say I didn't have a 1.12 section because there were no further steps for 1.12, but it made me realize there were no steps about `:inputs`.

What do you think of the updated docs for installation?

https://github.com/surface-ui/surface_formatter/pull/50/commits/6a2e96c38515656c864e4736f0c2e27f620086ef#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L11-R44

Reply:
that looks good!

Reply:
@lnr0626 Ok thanks, speak up though if you think it'd be clearer to separate them altogether for 1.12 vs 1.13.